bin/run-tests.sh

```diff
@@ -109,25 +111,43 @@ if [ "$CLEANUP_CONTAINERS_ON_EXIT" = "true" ]; then
     trap cleanup EXIT
 fi
 
+DOCKER_VERSION=`docker --version | sed -e 's/^.* version \([0-9.]*\).*$/\1/g'`
 if ! docker build --rm=$NO_DOCKER_CACHE --no-cache=$NO_DOCKER_CACHE -t marathon-buildbase:$BUILD_ID \
-    -f "$PROJECT_DIR/Dockerfile.build-base" "$PROJECT_DIR"; then
+    --build-arg DOCKER_VERSION=$DOCKER_VERSION -f "$PROJECT_DIR/Dockerfile.development" "$PROJECT_DIR"; then
     fatal "Build for the buildbase failed" >&2
 fi
 
-docker run \
-    --rm=$CLEANUP_CONTAINERS_ON_EXIT \
-    --name marathon-itests-$BUILD_ID \
-    --memory 4g \
-    --memory-swap 6g \
-    -e MARATHON_MAX_TASKS_PER_OFFER=$MARATHON_MAX_TASKS_PER_OFFER \
-    -v "$SBT_DIR:/root/.sbt" \
-    -v "$IVY2_DIR:/root/.ivy2" \
-    -v "$TARGET_DIRS/main:/marathon/target" \
-    -v "$TARGET_DIRS/project:/marathon/project/target" \
-    -v "$TARGET_DIRS/mesos-simulation:/marathon/mesos-simulation/target" \
-    -i \
-    marathon-buildbase:$BUILD_ID \
-    bash -c '
-    sbt -Dsbt.log.format=false test integration:test &&\
-    sbt -Dsbt.log.format=false "project mesos-simulation" integration:test "test:runMain mesosphere.mesos.scale.DisplayAppScalingResults"' \
-    || fatal "build/tests failed"
+docker_args=(
+    run
+    --rm=$CLEANUP_CONTAINERS_ON_EXIT
+    --name marathon-itests-$BUILD_ID
+    --memory 4g
+    --memory-swap 6g
+    --net host
+    -e "BUILD_ID=$BUILD_ID"
+    -e "IVY2_DIR=$IVY2_DIR"
+    -e "PROJECT_DIRS=$PROJECT_DIRS"
+    -e "SBT_DIR=$SBT_DIR"
+    -e "TARGET_DIRS=$TARGET_DIRS"
+    -e "MARATHON_MAX_TASKS_PER_OFFER=$MARATHON_MAX_TASKS_PER_OFFER"
+    -v "$SBT_DIR:/root/.sbt"
+    -v "$IVY2_DIR:/root/.ivy2"
+    -v "$TARGET_DIRS/main:/marathon/target"
+    -v "$TARGET_DIRS/project:/marathon/project/target"
+    -v "$TARGET_DIRS/mesos-simulation:/marathon/mesos-simulation/target"
+    -v "/etc/hosts:/etc/hosts"
+    --entrypoint /bin/bash
+    -i
+)
+
+if [ "$MOUNT_DOCKER_SOCKET" = "true" ]; then
+    docker_args+=(-v /var/run/docker.sock:/var/run/docker.sock)
+fi
+
+docker_args+=(
+    "marathon-buildbase:$BUILD_ID"
+    -c 'sbt -Dsbt.log.format=false test integration:test && sbt -Dsbt.log.format=false "project mesos-simulation" integration:test "test:runMain mesosphere.mesos.scale.DisplayAppScalingResults"'
```

Comment:
I know that it was like this before but it would be nice to break this very long line into multiple lines.